### ISSUE TITLE
Version is still 0.4.2 in 1.1.0-rc1 metadata

### ISSUE THREAD

**Issue by musicinmybrain** [ohsu-comp-bio/py-tes#59]
As of 1.1.0-rc1, the version in the package, which ends up in the wheel metadata, is still at 0.4.2.

https://github.com/ohsu-comp-bio/py-tes/blob/791352608794ef590ad13a078fbe5cd57da0e3e5/tes/__init__.py#L39

Please make sure this matches the actual version for the 1.1.0 final release. Thanks!

**Comment by lbeckman314** [comment]
Good catch! Just updated the version in the `release-1.1.0` branch to match that on PyPi.

https://github.com/ohsu-comp-bio/py-tes/blob/4d12b82ff89105fb42018af09926f14b3c51bce1/tes/__init__.py#L39

Will keep this value in sync in future version bumps, and update `master` and `develop` branches when the release branch is merged.

Thanks for finding this!
